dietstory-routing/login_bonus/serializers.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import serializers
+from .models import LoginBonusRewards, LoginBonus
+
+
+class LoginBonusRewardSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = LoginBonusRewards
+        fields = ('reward_num', 'reward_month', 'item_id', 'item_name', 'quantity', 'time_to_expire')
+
+
+class LoginBonusSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = LoginBonus
+        fields = ('reward_num', 'reward_month', 'latest_reward_time')
```

Comment:
I believe we specified to include a `reward_year` so that we cover the case over yearly cycles

Reply:
This was actually added.